### PR TITLE
New donor emails + more data modeling helper functions

### PR DESCRIPTION
**Pull request by lucashu1**
TODO:
1. Rewrite Donation Summary table so that it doesn't use flexbox (which isn't supported by most email clients, including Gmail) - Done!
2. Refactor `sqlHelpers` so that we use the same data models as described in the API documentation. I.e. move away from SQL column names (e.g. `sqlRow.donation_amt_usd`) to only using the API-style object specs (`donationObj.donationAmtUsd`). Probably do this in a future deliverable, maybe this coming week?

### CODE REVIEW

**Comment by lucashu1** [comment]
Screenshots from gmail (looks similar on mobile):
Screen Shot 2019-10-14 at 10 25 19 AM
Screen Shot 2019-10-14 at 10 25 48 AM

File: src/server/util/itemHelpers.js [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
-function getFrontEndItemObj(row) {
+function sqlRowToItemObj(row) {
   // SQL row to item object
   let itemObj = {
     itemId: row.item_id,
     image: row.item_photo_link,
     name: row.item_name,
     size: row.size,
-    price: row.price_euros,
+    price: row.price_euros.toFixed(2),
```
Comment (Raghav-Maheshwari) [inline src/server/util/itemHelpers.js]: if its not too much to change, maybe we should have this as `price_usd` just for more clarity

File: src/server/util/sqlHelpers.js [hunk 2]
```diff
@@ -471,7 +509,30 @@ async function getItems(itemIds) {
...
+      throw new Error(`getItemObjFromItemId: item not found for itemId: ${itemId}`);
+    }
+    return itemHelpers.sqlRowToItemObj(row);
+  } catch (err) {
+    errorHandler.handleError(err, "sqlHelpers/getItemObjFromItemId");
+    throw err;
+  }
+}
+
+async function getItemObjsFromItemIds(itemIds) {
```
Comment (Raghav-Maheshwari) [inline src/server/util/sqlHelpers.js]: Any way we can consolidate these two functions into one?

Comment (lucashu1) [reply src/server/util/sqlHelpers.js]: Will do in a future PR (with the refactoring)

**Pull request merged**